src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+  flex-direction: row;
+
+  &__star {
+    margin-right: 4px;
+    @include starSize($star-size);
+
+    background-image: url(../images/star.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+  }
+
+  @for $i from 1 through 5 {
```

Comment:
I think to name variables more clearly is better for understanding what's going on here